tests/coverage_analysis.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 
 set -e
-
 readonly libpath="${LIBPATH:-../guibot}"
 readonly coverage="${COVERAGE:-coverage3}"
 readonly submit="${SUBMIT:-0}"
 
+echo " ---- running tests with $coverage ---- "
+
```

Comment:
What is the reason for the empty line after the header? It seemed to me that we never add empty lines after headers so far.